apps/voting/app/src/script.js

```diff
@@ -169,12 +219,39 @@ async function startVote(state, { creator, metadata, voteId }) {
   }))
 }
 
+async function getAccountVotes({ connectedAccount = '', votes = [] }) {
+  const connectedAccountVotes = await Promise.all(
+    votes.map(({ voteId }) =>
+      getAccountSpecificVote({ connectedAccount, voteId })
+    )
+  )
+    .then(voteStates =>
+      voteStates.reduce((states, { voteId, voteType }) => {
+        states[voteId] = voteType
+        return states
+      }, {})
+    )
+    .catch(console.error)
+
+  return connectedAccountVotes
+}
+
+async function getAccountSpecificVote({ connectedAccount = '', voteId = '' }) {
+  return app
+    .call('getVoterState', voteId, connectedAccount)
+    .toPromise()
+    .then(voteTypeFromContractEnum)
+    .then(voteType => ({ voteId, voteType }))
+    .catch(console.error)
+}
```

Comment:
Could we move both of these new functions to be in the `helpers` section, since they're not directly called on events?